docs/messaging/messaging-spans.md

```diff
@@ -261,6 +261,23 @@ batch of messages, or for no message at all (if it is signalled that no
 messages were received). For each message it accounts for, the "Process" or
 "Receive" span SHOULD link to the message's creation context.
 
+> [!IMPORTANT]
+> These conventions use spans links as the default mechanism to correlate
+> producers and consumer(s) because:
+>
+> - It is the only consistent trace structure that can be guaranteed,
+> given the many different messaging systems models available.
+>
+> - It is the only option to correlate produce and consumer(s) in batch scenarios
+> as a span can only have a single parent.
+>
+> - Offers a consistent experience for users analysing traces from different
```

Comment:
It seems very similar to the first point, no?

> - It is the only consistent trace structure that can be guaranteed, given the many different messaging systems models available.

Reply:
Yeah good point. I tried to merge them together but didn't sound good. I will just remove it then :)